### ISSUE TITLE
Tokenizer return nothing instead of unk for certain token? 

### ISSUE THREAD

**Issue by zhouhanxie** [huggingface/transformers#8411]
## Environment info
     
- `transformers` version: 3.1.0
- Platform: pytorch
- Python version: python 3.6.9
- PyTorch version (GPU?):  1.6.0
- Tensorflow version (GPU?): na
- Using GPU in script?: no
- Using distributed or parallel set-up in script?: no

## Information

Model I am using (tokenizer && model == google/electra-large-generator):

The problem arises when using:
* [x ] my own modified scripts: (give details below)
self.fill_mask  = pipeline("fill-mask", model="google/electra-large-generator",\
                                              tokenizer="google/electra-large-generator")

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x ] my own task or dataset: (give details below)
a simple fill mask type of task with transformer pipline

## To reproduce

1. fill_mask  = pipeline("fill-mask", model="google/electra-large-generator",\
                                              tokenizer="google/electra-large-generator")
2. fill_mask.tokenizer.tokenize(""" ̈  """)

3. output is []

## Expected behavior

In this case the missing of this punctuation would cause some downstream usage to fail due to IndexError. The problem might be intrinsic to this particular tokenizer, maybe it is worthwhile to raise a warning/error or returning a unk token? Thanks!

**Comment by zhouhanxie** [comment]
Similar error at [1662]https://github.com/huggingface/transformers/issues/1662

The problem could be solved by switching to multilingual tokenizer(if present), else it would require some hot fix.
